superset-frontend/src/components/TableView/TableView.test.tsx

```diff
@@ -191,3 +191,12 @@ test('should render the right page', () => {
   expect(screen.getByText('Kate')).toBeInTheDocument();
   expect(screen.queryByText('Emily')).not.toBeInTheDocument();
 });
+
+test('should render the right wrap content text by columnsForWrapText', () => {
+  const props = {
+    ...mockedProps,
+    columnsForWrapText: ['Name'],
+  };
+  render(<TableView {...props} />);
+  expect(screen.getByTestId('table-row-cell__wrap')).toBeInTheDocument();
```

Comment:
This assertion could be a bit enhanced a bit (and the data-test attribute removed) if you did something like:
```js
expect(within(screen.getByTestId('table-row-table-row')).getAllByRole('cell')[3])toHaveClass('table-cell__wrap');
expect(within(screen.getByTestId('table-row-table-row')).getAllByRole('cell')[2])toHaveClass('table-cell__nowrap');
```